mgcpy/mgc/dcorr.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
```

Comment:
This file should not be under `mgc`, pull it out into a separate module under `mgcpy`.